ui/src/app/admin/_components/personTable.tsx

```diff
@@ -0,0 +1,284 @@
+                                            {cell.column.id === 'inBasisAndInclude' && (
+                                                <div>
+                                                    <p
+                                                        className={`${
+                                                            cell.row.original.inBasisAndInclude ? 'text-red-500' : ''
+                                                        }`}
+                                                    >
+                                                        {cell.row.original.inBasisAndInclude ? 'Yes' : 'No'}
+                                                    </p>
+                                                </div>
+                                            )}
+                                            {cell.column.id === 'inInclude' && (
+                                                <div className="ml-2">
+                                                    <p
+                                                        className={`${
+                                                            cell.row.original.inInclude ? 'text-red-500' : ''
+                                                        }`}
+                                                    >
+                                                        {cell.row.original.inInclude ? 'Yes' : 'No'}
+                                                    </p>
+                                                </div>
+                                            )}
+                                            {cell.column.id === 'inExclude' && (
+                                                <div className="ml-2">
+                                                    <p
+                                                        className={`${
+                                                            cell.row.original.inExclude ? 'text-red-500' : ''
+                                                        }`}
+                                                    >
+                                                        {cell.row.original.inExclude ? 'Yes' : 'No'}
+                                                    </p>
+                                                </div>
+                                            )}
+                                            {cell.column.id === 'remove' && (
+                                                <div className="ml-3">
+                                                    <input
+                                                        type="checkbox"
+                                                        checked={row.getIsSelected()}
+                                                        onChange={row.getToggleSelectedHandler()}
+                                                    />
+                                                </div>
+                                            )}
```

Comment:
These && statements should be moved to the person-table-columns.tsx file. Reference how the groupings-table-columns.tsx file does it.